### ISSUE TITLE
Create a Device Profile only gives the option to Cancel. 

### ISSUE THREAD

**Issue by kiakri** [Adelenade/GoXlr-Macos#20]
**Describe the bug**

**To Reproduce**

Load Profile > Device Profile > Load profile / New Profile > only option is to cancel. 

**Expected behavior**
Should be able to create and load a profile.

**Screenshots**
![image](https://user-images.githubusercontent.com/7399660/195869012-f1a27962-05cf-46f8-8052-1ce9aafa3e68.png)

**Desktop (please complete the following information):**
 - Device [e.g. MacBook pro M1 max] - M1 Max
 - GoXlr model [e.g. full, mini] Full
 - Version of the app [e.g. 0.3.3] - V1 Alpha 3

**Additional context**

**Comment by Adelenade** [comment]
Hi ! This bug has already beed reported on discord, and as i said there, it's a problem with SwiftUI itself.
In the code there is:

- The text field
- The cancel button
- The confirm button

One of these two buttons are the "confirm" button, but SwiftUI is bugging and show the confirm button as it was the cancel one.
If you have a touchbar, youll notice that the cancel and confirm button are here.
Meanwhile, im going to see if i can make multiple alerts like this, as having only two elements on an alert don't bug (don't know why) !

**Comment by kiakri** [comment]
Yeah so when clicking on either of them a profile isn't generated at all. It just does nothing at all for me.
